docker/Makefile

```diff
@@ -1,5 +1,6 @@
 PLATFORMIO_VERSION?=4.1.0
 QEMU_COMMIT?=esp-develop
+DEV=$(shell dmesg | awk '/cp210x/ && /ttyUSB/ { print $$NF }')
```

Comment:
change to `DEV?=`

Reply:
is sth like `readlink -f /dev/serial/by-id/usb-Silicon_Labs_CP2102_USB_to_UART_Bridge_Controller*` maybe better?

Reply:
I would say the check for cp210x is more generic, but i would need to test some more different devices to be sure

Reply:
Ok, this version wont work if a ttyUSB was connected and removed later. It will still show up in dmesg, but isn't present in /dev/ anymore.

Reply:
how about adding DEV variable to make to let user specify (esp. with several ttyUSB devices) which devices to use?

Reply:
I don't think the reconnect issue would lead to any major problems, at least not in the current state.

Reply:
your wish for a generic solution is also possible via globbing `readlink -f /dev/serial/by-id/*CP210*`. To be honest, I just dislike the idea of "parsing" `dmesg`

Reply:
> how about adding DEV variable to make to let user specify (esp. with several ttyUSB devices) which devices to use?

you already achieved this by adding the DEV variable, see my comment on the `platformio.ini`